src/components/Blog/Blog.module.css

```diff
@@ -0,0 +1,59 @@
+.contentBlogWrapper {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: space-between;
+  & > div {
+    flex: 1;
+    margin: 2rem;
+    background-color: #271b96;
+  }
+}
+.item {
+  transition: transform 0.3s ease;
+}
+
+.item:hover {
+  transform: scale(1.05);
+}
+
+.contentShopFounder {
+  background-size: cover;
+  width: 100%;
+  height: 100%;
+  background-image: url(../../assets/Marcin.jpg);
```

Comment:
replace all of images with img tags in code - it will enable you to display data when will come from APi